### DISCUSSION TITLE
The Action Wheel Is Still Empty

### DISCUSSION THREAD

**Discussion by ArthurtheBud** [ErinaSugino/Starbound-SBR-Mod#75] [Trouble Shooting] [answered]
I was playing with a female character. When I tried to interact with sexnodes with either an NPC or a player, i couldn't select animations, and when i left it, the sprite stayed there. So was the music. I've removed the Lustbound base, Lustbound base manager, Blake's Terramons, the patches related to Blake's Terramons, and changed the value of forceAllPositions in the main config to true. Same thing still occured. 

[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/13463744/starbound.log)

How do I fix this?

**Comment by ErinaSugino** [comment, marked as answer]
You have `Lustbound Base TentacleTweak` installed. An *even older and even more broken* version of Lustbound Base.

**Reply by ArthurtheBud** [reply]
I feel like a donkey now....
well, gotta need a better glasses. It's fixed. Thank you very much!
